pybase/region/client.py

```diff
@@ -64,8 +64,8 @@ class Client(object):
     #   we can match incoming responses with the rpc that made the request.
 
     def __init__(self, host, port, secondary):
-        self.host = host
-        self.port = port
+        self.host = host.decode('utf8') if isinstance(host, bytes) else host
+        self.port = port.decode('utf8') if isinstance(port, bytes) else port
```

Comment:
even though `port` works fine in most places as a byte string, let's make it a unicode string to make our lives easier elsewhere